ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/ethernet-interfaces.md

```diff
@@ -1175,6 +1175,46 @@ interface Ethernet72
    dot1x aaa unresponsive action traffic allow vlan 10 access-list acl1
    dot1x aaa unresponsive eap response success
    dot1x mac based access-list
+!
+interface Ethernet73
+   description Switchport_tap_tool
+   switchport tap native vlan 10
+   switchport tap identity 3 inner 5
+   switchport tap mac-address dest 01:00:00:00:00:00 src 01:23:45:67:89:ab
+   switchport tap mpls pop all
+   switchport tap encapsulation gre strip
+   switchport tap encapsulation gre destination 1.1.1.1 source 1.1.1.2 protocol 0x0000 strip
+   switchport tap encapsulation gre protocol 0x0001 strip
+   switchport tap encapsulation gre destination 2.1.1.2 protocol 0x0010 strip
+   switchport tap encapsulation gre destination 2.1.1.3 source 2.1.1.4 strip
```

Comment:
'''s1-leaf1(config-if-Et2)#switchpor tap header remove size 10'''
This command is also supported under tap configuration.

Reply:
The command does not reflect on running-config